col/src/main/java/vct/col/ast/expr/StandardOperator.java

```diff
@@ -2,10 +2,6 @@
 package vct.col.ast.expr;
 
 public enum StandardOperator {
-  /** get a location */
-  Get(1),
-  /** set a location (and return new value) */
-  Set(2),
```

Comment:
Oof, I don't really remember... In any case I think it's duplicate with `FieldAccess` (which in turn is only used for futures/histories). There was also some idea to type writable expressions (say a simple name `x`) as `location<T>` instead of just `T`, maybe that was what `Get` was intended for (seeing as it has only 1 argument).